configure

```diff
@@ -16,10 +16,11 @@ fi
 ## Parse the args, only looking for -camldir
 ## We avoid using shift to keep "$@" intact
 
+cmd=$ocaml
 last=
 for i; do
    case $last in
-       -camldir|--camldir) cmd="$i/ocaml"; break;;
```

Comment:
Indeed I mixed up `configure` and `coqtop`. Sorry for the noise.

> Or do you mean that all long-style options should exist with -- too? This would indeed be the most flexible and open-minded approach, while still satisfying uniformity.

This would indeed be the approach I would prefer but this is not a strong opinion.